connector/oidc/oidc.go

```diff
@@ -107,6 +107,9 @@ type ProviderDiscoveryOverrides struct {
 	// AuthURL provides a way to user overwrite the Auth URL
 	// from the .well-known/openid-configuration authorization_endpoint
 	AuthURL string `json:"authURL"`
+	// JWKSURL provides a way to user overwrite the JWKS URL
+	// from the .well-known/openid-configuration jwks_uri
+	JWKSURL string `json:"jwksURL"`
 }
 
 func (o *ProviderDiscoveryOverrides) Empty() bool {
```

Comment:
Pls do a check for JWKSURL as well, current condition only checks authurl and tokenurl to override values. This will not work if only JWKSURL is passed. 
Add required changes to [oidc_test.go](https://github.com/dexidp/dex/blob/master/connector/oidc/oidc_test.go) for consistency.